tests/sss/test_secrets.py

```diff
@@ -16,5 +16,11 @@ def test_secrets(addon_with_secrets):
             )
             == no_of_secrets
         )
+        for secret in sss_walker["template_deploy"]["objects"][0]["spec"][
+            "resources"
+        ]["Secret"]:
+            assert secret[1]["metadata"]["labels"] is not None
```

Comment:
```suggestion
            assert secret[1].get("metadata", {}).get("labels") is not None
```
otherwise if metadata or labels is not found, Python would raise KeyError instead of just returning None.